frontend-ssr/components/exercise-submissions/SubmissionSkeleton.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styles from './SubmissionSkeleton.module.scss';
+import { Submission, SubmissionType } from '~/redux/exercise-submissions/types';
+
+interface Props {
+  submission: Submission,
+}
+export default function SubmissionSkeleton({ submission } : Props) {
```

Comment:
It seems the markup of this component is pretty much the same as the main component. This shouldn't be the case at all. 

A skeleton component should be much much MUCH simpler: just a couple of placeholder elements without any real content. So this should be refactored and have just empty elements without any props or content.